docs/specs/casing.yml

```diff
@@ -1,5 +1,4 @@
-# Path is case insensitive on windows and mac
```

Comment:
is that a break change in v3?

Reply:
if we want to all are case insensitive, we need re-design it, a lot of comparing logic is also applied too.

Reply:
sure, defer that change